Packs/CommonScripts/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.14.3",
+    "currentVersion": "1.15.0",
```

its a minor update, please update the rn file name as well.
```suggestion
    "currentVersion": "1.14.4",
```